src/script/ceph-backport.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash -ex
+#
+# ceph-backport.sh
+#
+# Ceph backporting script
+#
+# Assumes you have forked ceph/ceph.git, cloned your fork, and are running the
+# script in the local clone!
+#
+# With this script, backporting workflow for backport issue
+# http://tracker.ceph.com/issues/19206 (a jewel backport)
+# becomes something like this:
+#
+# git remote add ceph http://github.com/ceph/ceph.git
+# git fetch ceph
+# git checkout -b wip-19206-jewel ceph/jewel
+# git cherry-pick -x $MASTER_COMMIT_SHA1
```

Comment:
I assume `MASTER_COMMIT_SHA1` is the merge commit? Does that actually work? I've always used

`"${MASTER_MERGE_COMMIT}^..${MASTER_MERGE_COMMIT}^2"` which is guaranteed to get all of the commits in the PR.

Reply:
The choice of cherry-picking method is orthogonal to the script, I guess. I'll rephrase that line.

(I always cherry-pick each commit individually, since I never know which one will have a conflict. And no, I don't think `git cherry-pick -x $MERGE_COMMIT` works, but I guess I haven't really tried it?)